File: src/main/java/xyz/mattyb/openwest/kotlinprimer/chapter02/Strings.java
package xyz.mattyb.openwest.kotlinprimer.chapter02;

public class Strings {
    public static void main(String[] args) {
        String firstName = "matt";
        String lastName = "berteaux";
        String name = firstName.substring(0, 1).toUpperCase() + firstName.substring(1) + " "
                + lastName.substring(0, 1).toUpperCase() + lastName.substring(1);

        System.out.println(name);

        String haiku = "Five syllables here.\n" +
                "Seven more syllables here.\n" +
                "Are you happy now?";

        System.out.println("\""+ haiku +"\"");
    }
}

File: src/main/java/xyz/mattyb/openwest/kotlinprimer/chapter02/strings.kt
package xyz.mattyb.openwest.kotlinprimer.chapter02

fun main(args: Array<String>) {
    /**
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     * Don't want to give away how functions work ;-)
     */
    val firstName = "matt"
    val lastName = "berteaux"
    val name = "${firstName.capitalize()} ${lastName.capitalize()}"
    println(name)

    val haiku = """
        |Five syllables here.
        |Seven more syllables here.
        |Are you happy now?
    """.trimMargin()

    println("\"$haiku\"")
}